compiler/passes/normalize.cpp

```diff
@@ -4588,6 +4588,7 @@ static void find_printModuleInit_stuff() {
     if (symbol->hasFlag(FLAG_PRINT_MODULE_INIT_INDENT_LEVEL)) {
       gModuleInitIndentLevel = toVarSymbol(symbol);
       INT_ASSERT(gModuleInitIndentLevel);
+      // Q why  doesn't this break out of the loop
```

Comment:
🤔

Reply:
I don't know if you want to try adding a `break` and then running tests, but I can't imagine we have more than one symbol with that flag.

Reply:
Would you prefer I try to tackle and answer that now? I left it rhetorical with the intention of addressing it when attacking normalize.cpp, but could investigate further now too

Reply:
Oh are you planning on catching that variable during normalize or something instead? I'm fine with either way, I'm not confident a `break` will make much of a difference (though I do think it's appropriate).

Reply:
I pushed a commit that answers this question. It didn't turn out to be complicated so I'm including it in this PR

Reply:
Ah, ok, this makes sense. Thanks for taking the time to answer this!

Reply:
Thanks for the push to answer it!